hyo2/openbst/lib/processing/parameters.py

```diff
@@ -0,0 +1,26 @@
+import logging
+
+from hyo2.openbst.lib.processing.process_methods.dicts import ProcessMethods
+from hyo2.openbst.lib.processing.process_methods.raw_decoding import RawDecodeParameters
+from hyo2.openbst.lib.processing.process_methods.static_gain_compensation import StaticGainParameters
+from hyo2.openbst.lib.processing.process_methods.source_level import SourceLevelParameters
+
+logger = logging.getLogger(__name__)
+
+
+class Parameters:
+    """Class to store processing parameters"""
+
+    def __init__(self):
+        self.rawdecode = RawDecodeParameters()
+        self.static_gains = StaticGainParameters()
+        self.source_level = SourceLevelParameters()
+
+    # TODO: Can there be a check here to create object if not yet created
+    def get_process_params(self, process_type: ProcessMethods):
```

Comment:
What is the use of this get function given that `rawdecode`, `static_gains`, and `source_level` are public class variables?